### PR TITLE
Introduced XML Resource Repository Differ

### LINKED ISSUE #4711: [Feature Request] Extend Differ to compare OBR XMLs
In many scenarios, people require to compare OBR XMLs directly but the current implementation supports only comparison between JARs comprising OBR XMLs. It would be really useful to extend the current implementation of `aQute.bnd.differ.DiffPluginImpl` for the support of OBR XML comparison.

### PR DESCRIPTION
**Pull request by amitjoy**
Fixes #4711

### CODE REVIEW

File: biz.aQute.bnd.util/src/aQute/bnd/unmodifiable/Maps.java [hunk 1]
```diff
@@ -62,6 +62,12 @@ private Maps() {}
 			entry(k6, v6), entry(k7, v7), entry(k8, v8), entry(k9, v9), entry(k10, v10));
 	}
 
+	public static <K, V> Map<K, V> of(K k1, V v1, K k2, V v2, K k3, V v3, K k4, V v4, K k5, V v5, K k6, V v6, K k7,
```
Comment (bjhargrave) [inline biz.aQute.bnd.util/src/aQute/bnd/unmodifiable/Maps.java:65]: Lets not do this. Just use the existing methods like ofEntries.

Comment (amitjoy) [reply biz.aQute.bnd.util/src/aQute/bnd/unmodifiable/Maps.java]: DONE

**Comment by pkriens** [comment]
@amitjoy will you fix this baseline error?

**Comment by amitjoy** [comment]
@pkriens I have just bumped the versions

**Pull request merged**